clang/lib/Sema/SemaChecking.cpp

```diff
@@ -755,6 +755,7 @@ static ExprResult BuiltinDumpStruct(Sema &S, CallExpr *TheCall) {
     case BuiltinType::PseudoObject:
     case BuiltinType::UnknownAny:
     case BuiltinType::BuiltinFn:
+    case BuiltinType::BuiltinCountedByRef:
```

Comment:
```suggestion
```
`__builtin_counted_by_ref` isn’t a valid argument for `__builtin_dump_struct`, so this should fall under the default case below.

Reply:
Then shouldn't *all* builtin functions be marked as "invalid?"